deployment/modules/github/org/secrets.tf

```diff
@@ -51,3 +51,15 @@ resource "github_actions_organization_secret" "push_o_matic_app_key" {
   plaintext_value = data.onepassword_item.push_o_matic_app.private_key
   visibility      = "all"
 }
+
+resource "github_actions_organization_secret" "docker_hub_read_token" {
+  secret_name     = "DOCKER_HUB_READ_TOKEN"
+  plaintext_value = data.terraform_remote_state.docker_org_state.outputs.read_token
+  visibility      = "all"
+}
```

Comment:
Doing the docker hub auth in GHA isn't compatible with fork runs, so we reverted that and I set up the pull through cache on mich. I expect that should be fine even without auth but if you want we can send this token that way instead.

Reply:
Yea I saw that, I figured having read and write tokens available means if we need them in future they are available with less effort, allowing people to use a read token where applicable.